pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -696,17 +699,11 @@ func (c *Cluster) getMonPlacement(zone string) cephv1.Placement {
 }
 
 func realWaitForMonitorScheduling(c *Cluster, d *apps.Deployment) (SchedulingResult, error) {
-	ctx := context.TODO()
 	// target node decision, and deployment/pvc to cleanup
 	result := SchedulingResult{}
 
 	// wait for the scheduler to make a placement decision
 	for i := 0; i < canaryRetries; i++ {
-		// Check whether we need to cancel the orchestration
-		if err := controller.CheckForCancelledOrchestration(c.context); err != nil {
```

Comment:
I thought you said we are still using checkpoints. Where are the checkpoints? This one must be covered by cancelling the context since it's removed.

Reply:
Sorry about that, this goes lost during the change. Adding back.